rocAL/rocAL/include/pipeline/tensor.h

```diff
@@ -138,7 +138,8 @@ class TensorInfo {
         }
     }
     void set_tensor_layout(RocalTensorlayout layout) {
-        if(_layout != layout && _layout != RocalTensorlayout::NONE) {
+        if (layout == RocalTensorlayout::NONE) return;
+        if (_layout != layout && _layout != RocalTensorlayout::NONE) {  // If layout input and current layout's are different modify dims accordingly
```

Comment:
remove the second condition for if which is implied because of line #141

Reply:
The first condition checks if the input layout is None
Whereas the second condition checks if the current layout of the tensor is None